hedera-node/src/main/java/com/hedera/services/state/merkle/MerkleStakingInfo.java

```diff
@@ -49,7 +50,6 @@ public class MerkleStakingInfo extends AbstractMerkleLeaf implements Keyed<Entit
 	private static final Logger log = LogManager.getLogger(MerkleStakingInfo.class);
 	// get this max history from the props.
 	static final int MAX_REWARD_HISTORY = 366;
```

Comment:
Unused?

Reply:
Will be used for deserialization. need it a max array size limit while reading the reward sum history array from the serializable input stream.